warehouse/query-core/src/test/java/datawave/query/predicate/TLDEventDataFilterTest.java

```diff
@@ -594,4 +590,18 @@ public void apply_acceptGroupingFields() throws ParseException {
         assertFalse(result3);
         assertTrue(result4);
     }
+    
+    @Test
+    public void testBadIdentifier() throws ParseException {
```

Comment:
Would it make sense to add a similar test to this with an identifier that had mixed capitalization in it, to confirm that those fields would be rejected as well?

Reply:
Can do